README.markdown

```diff
@@ -273,43 +254,21 @@ listToMaybe []
 
 <https://begriffs.com/posts/2013-08-18-dont-be-partial-to-partial-functions.html>
 
-## Avoid throwing exceptions
-
-Even if you're already in `IO`, you should prefer expressing failure with types over using `throw`.
-Just because `IO` can throw exceptions doesn't mean that the only failure mode in `IO` should be exceptions.
-
-``` hs
--- bad
-if canAccess thing user
-  then pure thing
-  else throw (userError "cannot access thing")
-
--- good
-if canAccess thing user
-  then pure (Just thing)
-  else pure Nothing
-```
-
-Throwing exceptions is fine if the caller can't do anything about it anyway.
-You shouldn't throw exceptions that you expect callers to catch.
-
-<https://np.reddit.com/r/haskell/comments/5bkqf1/exceptions_best_practices_in_haskell/>
+## Group all imports together
 
-## Group imports together
-
-Imports should be split into two groups: third-party and first-party.
+Imports should not be split into two groups: third-party and first-party.
```

Comment:
This is a big change 😄 Should we update our codebase first before we update the style guide?

Reply:
https://github.com/EdutainmentLIVE/smurf/pull/3021 done!